components/create_comment/index.js

```diff
@@ -27,10 +28,12 @@ function mapStateToProps(state, ownProps) {
     const draft = getCommentDraft(state);
 
     const enableAddButton = draft.message.trim().length !== 0 || draft.fileInfos.length !== 0;
+    const channelMembersCount = getAllChannelStats(state)[ownProps.channelId] ? getAllChannelStats(state)[ownProps.channelId].member_count : 1;
```

Comment:
We can replace the getAllChannelsStats(state)[ownProps.channelId) creating a new selector in mattermost-redux with the signature getChannelStats(state, channelId). But for simplicity I do it in this way. If you prefer in can move it to the mattermost-redux repo.